website/src/controllers/groups.js

```diff
@@ -211,7 +211,6 @@ api.create = function(req, res, next) {
       if (err == 'Already in a party, try refreshing.') return res.json(400,{err:err});
       if (err) return next(err);
       return res.json(populated);
-      group = user = null;
```

Comment:
I think these were meant to be forced garbage collection for our memory leak problem. Can these be moved to a location where they'll take place, rather than removed entirely?